blueman/plugins/applet/KillSwitch.py

```diff
@@ -35,9 +35,8 @@ def __init__(self, idx, switch_type, soft, hard):
 
 class KillSwitch(AppletPlugin):
     __author__ = "Walmis"
-    __description__ = _("Toggles a platform Bluetooth killswitch when Bluetooth power state changes "
-                        "(Useless with USB dongles) and makes sure a status icon is shown if there is a bluetooth "
-                        "killswitch but no adapter.")
+    __description__ = _("Switches Bluetooth killswitch status to match Bluetooth power state."
+                        "If the device is physically removed, a staus icon is shown.")
```

Comment:
Not 100% sure here.

Reply:
It's the other way around. The plugin makes sure that the icon is displayed if Bluetooth is soft-blocked by a killswitch so that the user can use it to re-enable it. Otherwise, the StatusIcon plugin would hide it as it cannot see any adapters and feels useless.

I guess "there is a bluetooth killswitch but no adapter" was meant to mean exactly that: Bluetooth soft-blocked by a killswitch an no adapter *visible by the system* (otherwise the icon would be shown anyway).